### PR TITLE
Add Claudy as a service

### PR DESCRIPTION
**Pull request by CPatchane**
Add Claudy menu service to be use as intent in the cozy-bar.

### CODE REVIEW

**Comment by CPatchane** [comment]
[(5) [NUA-3] ETU mes applications n'ont pas besoin de permissions pour afficher Claudy](https://trello.com/c/gHTic2pp/501-5-nua-3-etu-mes-applications-nont-pas-besoin-de-permissions-pour-afficher-claudy)

**Comment by lespacedunmatin** [comment]
There is a lint issue that blocks Travis.

File: src/locales/en.json [hunk 1]
```diff
@@ -115,5 +115,34 @@
...
+        "button": "Install the Cozy Drive app on your mobile",
+        "link": "https://docs.cozy.io/en/download/"
+      },
+      "cozy-collect": {
+        "title": "Get back your bills instantly",
+        "description": "Cozy Collect organize all your files instantly",
+        "button": "Discover Cozy Collect"
+      },
+      "support": {
+        "title": "How can we help you today ?",
```
Comment (lespacedunmatin) [inline src/locales/en.json]: s/today ?/today?/

**Comment by CPatchane** [comment]
Thanks @lespacedunmatin, that should be fixed 👍

**Pull request merged**